contributed_definitions/NXisocontour.nxdl.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@kuehbachm check if NXisocontour should be kept or if NXcg_isocontour is needed still and used somewhere?

Reply:
yes NXcg_isocontour was a duplicate of NXisocontour, result of consolidation work: NXcg_isocontour is removed, and all instances use now NXisocontour